cognite/client/_api/datapoints.py

```diff
@@ -84,7 +86,7 @@ def retrieve(
                 ...                         external_id={"externalId": "1", "aggregates": ["max"]},
                 ...                         start="1d-ago", end="now", granularity="1h")
         """
-        fetcher = DatapointsFetcher(client=self)
+        fetcher = DatapointsFetcher(client=self, ignore_unknown_ids=ignore_unknown_ids)
```

Comment:
To stay aligned with how we handle all other parameters, it should rather be an argument to `DatapointsQuery`.

Reply:
done